__tests__/app.test.js

```diff
@@ -354,3 +354,83 @@ describe("POST /api/articles/:articleid/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:articleid/", () => {
+  const moreVotes = {
+    inc_votes: 3,
+  };
+  const lessVotes = {
+    inc_votes: -2,
+  };
+  const invalidVotes = {
+    inc_votes: "banana",
+  };
+  const noVotes = {
+    colour: "green",
+  };
+  test("responds with status code 200 and the updated article when incrementing votes", () => {
+    return request(app)
+      .patch("/api/articles/4/")
+      .send(moreVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 4,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          author: "rogersop",
+          body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+          created_at: "2020-05-06T01:14:00.000Z",
+          title: "Student SUES Mitch!",
+          topic: "mitch",
+          votes: 3,
+        });
+      });
+  });
+  test("responds with status code 200 and the updated article when decrementing votes", () => {
+    return request(app)
+      .patch("/api/articles/1/")
+      .send(lessVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 1,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          votes: 98,
+        });
+      });
+  });
+  test("responds with a 404 code and appropriate error message when passed an article id that is not in the database", () => {
+    return request(app)
+      .patch("/api/articles/50/")
+      .send(moreVotes)
+      .expect(404)
+      .then(({ text }) => {
+        expect(text).toBe("No article found for article_id 50");
+      });
+  });
+  test("responds with a 400 code and appropriate error message when passed an article id that is not in the database", () => {
```

Comment:
You need to change your tests descriptions as the last three tests are the same but don't reflect the same behaviour.